### PR TITLE
Update README.md

### LINKED ISSUE #38: Code generation causes compilation failure for web
On bare template, after running (as per docs):

```bash
flutter_rust_bridge_codegen --rust-input native/src/api.rs --dart-output ./lib/bridge_generated.dart --dart-decl-output ./lib/bridge_definitions.dart
```

I get compilation failure for web. Happens even when using the same version of `flutter_rust_bridge_codegen` as the template. Might be related to the following lines disappearing from `bridge_generated.rs` ?:

```rust
/// cbindgen:ignore
#[cfg(target_family = "wasm")]
#[path = "bridge_generated.web.rs"]
mod web;
#[cfg(target_family = "wasm")]
pub use web::*;
```

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.7.1, on macOS 13.0 22A380 darwin-arm64, locale
    en-NZ)
[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 14.0)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2021.3)
[✓] VS Code (version 1.74.3)
[✓] Connected device (2 available)
[✓] HTTP Host Availability

• No issues found!
```

### PR DESCRIPTION
**Pull request by elliotsayes**
Closes #38 
Improved setup/run instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,14 +17,14 @@ To begin, ensure that you have a working installation of the following items:
 ```
 echo "ANDROID_NDK=.." >> ~/.gradle/gradle.properties
 ```
-
+- For iOS targets:
+  - Install [cargo-xcode](https://gitlab.com/kornelski/cargo-xcode#installation)
 - [Web dependencies](http://cjycode.com/flutter_rust_bridge/template/setup_web.html) for the Web
```
Comment (Desdaemon) [inline README.md:22]: Seems like this line was inadvertently deleted?

**Review by Desdaemon — CHANGES_REQUESTED** [review]
LGTM with nits

File: README.md (hunk 1 above)
Comment (elliotsayes) [reply README.md:23]: My bad!

**Comment by Desdaemon** [comment]
Thanks for contributing!

**Pull request merged**